src/ImageSharp/Formats/Webp/Lossy/QuantEnc.cs

```diff
@@ -77,9 +75,7 @@ public static void PickBestIntra16(Vp8EncIterator it, ref Vp8ModeScore rd, Vp8Se
 
                 if (mode == 0 || rdCur.Score < rdBest.Score)
                 {
-                    Vp8ModeScore tmp = rdCur;
-                    rdCur = rdBest;
-                    rdBest = tmp;
+                    (rdBest, rdCur) = (rdCur, rdBest);
```

Comment:
Use the swap-method from above (where I commented) instead (needs to be made generic then)?
Or drop the swap-method and write the one-line directly?

I don't have a strong opinion what's better here, but it should be consistent.
Leaning towards swap-method, as if there's a super nice trick once on how to swap more efficiently only one place needs to be updated.

Reply:
👍 good idea